### PR TITLE
Masterclass models

### PR DESCRIPTION
**Pull request by mforner13**
I've added a new MasterclassAttendees class with masterclass instance and user ids in it. 

To create the relationship between a user and a masterclass instance, I've removed the relationship between masterclass instance (when they're an attendee) and user, and just have the relationship between the user and the masterclass attendee table.

### CODE REVIEW

**Comment by mforner13** [comment]
@jonodrew please could you look at this pull request and check I've made the relationship between a user and a masterclass instance in the right way? 

I also wanted to check - when referencing a class attribute when the class has two words in camel case, would 'masterclassinstance.id' be correct?

**Comment by jonodrew** [comment]
> @jonodrew please could you look at this pull request and check I've made the relationship between a user and a masterclass instance in the right way?
> 
> I also wanted to check - when referencing a class attribute when the class has two words in camel case, would 'masterclassinstance.id' be correct?

Unfortunately not, SQLAlchemy converts your camelCase to snake_case so it should become `masterclass_instance.id`

File: app/models.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    def __repr__(self):
+        return '<User {} {}>'.format(self.first_name, self.last_name)
+
+
+class Masterclass(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(150))
+    description = db.Column(db.String(500))
+    timestamp = db.Column(db.DateTime, index=True)
+    masterclass_instances = db.relationship('MasterclassInstance', backref='location', lazy='dynamic')
```
Comment (jonodrew) [inline app/models.py]: I'm a little nervous about having two "masterclass_instances" methods on different classes. Are there other things you could name these?

**Review by jonodrew — COMMENTED** [review]
I think this looks really good. Without getting into the philosophy :nerd_face: , I wonder about the names of these classes. I would personally change the first class to `MasterclassContent` so it's more descriptive and the final class to `Masterclass` (renaming all the relationships). However, if you're happy with them as they are then I think they are technically correct which is, of course, the best kind of correct.

**Pull request closed without merging**